action.yml

```diff
@@ -15,6 +15,7 @@ runs:
   using: 'composite'
   steps:
   - name: Prepare Tutor and Launch
+    if: ${{ env.INPUT_TUTOR_VERSION != 'nightly' }}
```

Comment:
If it's possible to execute nightly directly with the `tutor local launch -I`, I think we can name this step like `Install Tutor` and unify the Tutor installations here